src/reporter/tests/test_NTNENA.py

```diff
@@ -1164,6 +1175,7 @@ def test_aggregation_is_per_instance(service, reporter_dataset):
     assert r.status_code == 200, r.text
 
     obtained = r.json()
+    delete_test_data(service, [entity_type])
```

Comment:
you can remove this, the yield in `reporter_dataset()` covers that

Reply:
removed